packages/projects-docs/pages/learn/plans/plan-overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Team and access management
+description: Manage your teams and control access to your workspace inside CodeSandbox Repositories
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Plans
+
+## Personal
+Everyone that signs up to CodeSandbox gets a free personal workspace. Here you can work on unlimited public sandboxes and repositories. If you want to work on anything private or work in collaboration with others, you will need a Pro subscription. 
+
+## Pro
+Pro offers unlimited collaboration, umlimited sandboxes and repositories (both private and public), higher VM specs and [much more](https://codesandbox.io/pricing).
+
+
+
+### Creating a new Pro worksapce
+
+1. Go to the [Dashboard](https://codesandbox.io/dashboard).
+1. Open the workspace selector and click on the `Create a Pro workspace` button at the bottom.
```

Comment:
Shouldn't we add here the Pro page selection. step 3 would be `Choose to start a trial or create a pro team based on your availability` or something like that